### ISSUE TITLE
ProtoOASubscribeSpotsRes could contain a repeated int64 symbolId = 3; // Unique identifier of the subscribed Symbol in cTrader platform.

### ISSUE THREAD

**Issue by raul-gherman** [spotware/Open-API-2.0-protobuf-messages#14]
// ProtoOASubscribeSpotsRes proposal
message ProtoOASubscribeSpotsRes {
    optional ProtoOAPayloadType payloadType = 1  [default = PROTO_OA_SUBSCRIBE_SPOTS_RES];
    required int64 ctidTraderAccountId = 2; // Unique identifier of the trading account. Used to match 
    responses to trading accounts.
    repeated int64 symbolId = 3; // Unique identifier of the subscribed Symbol in cTrader platform.
}

**Comment by LeoSko** [comment]
@raul-gherman, could you explain what for?

**Comment by raul-gherman** [comment]
@LeoSko,  notifying the underlying application about the array of subscribed symbols makes the Res less blunt and more informative, without being spammy

**Comment by LeoSko** [comment]
@raul-gherman, when sending request you can specify `ProtoMessage.clientMsgId` and later use it to match request with response. By the way, this works for any request-response pairs.

**Comment by raul-gherman** [comment]
correct, still that would mean one more req/res, which could be eliminated
less overhead on both ends

**Comment by LeoSko** [comment]
@raul-gherman, how does this mean one more req/res? There is only 1 request with `clientMsgId` with only 1 response with the same `clientMsgId`.

**Comment by raul-gherman** [comment]
i meant one more request to get the subscribed symbols, when they could actually be sent in ProtoOASubscribeSpotsRes

**Comment by LeoSko** [comment]
There is no request to "get the subscribed symbols".
You are supposed to
- send request for subscription `ProtoOASubscribeSpotsReq` with array of symbols you wish to subscribe `symbolId` with some random `clientMsgId` (which is specified in wrapper `ProtoMessage`)
- in response you get `ProtoMessage` with same `clientMsgId` and `ProtoOASubscribeSpotsRes` as payload, which means subscribe was successful for previously specified symbols

That flow assumes that you can manage current list of subscribed symbols in your application by yourself following successful/failure responses.
